projects/igniteui-angular/src/lib/core/styles/components/calendar/_calendar-theme.scss

```diff
@@ -795,7 +799,9 @@
     %cal-value--selected--hover {
         %cal-value-content {
             ::after {
-                background: rgba($selected-hover-color, .18);
+                @if type-of($selected-hover-color) == 'color' {
+                    background: rgba($selected-hover-color, .18);
```

Comment:
We should still set the background to whatever the user has passed as value. Add an else clause that assigns the value directly to the passed user value.